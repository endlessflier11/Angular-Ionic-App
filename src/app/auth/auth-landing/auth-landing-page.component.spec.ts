import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { AuthLandingPage } from './auth-landing-page.component';
import { PageTestingModule } from '@app/testing';

describe('AuthLandingPage', () => {
  let component: AuthLandingPage;
  let fixture: ComponentFixture<AuthLandingPage>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [AuthLandingPage],
      imports: [PageTestingModule.withConfig({ providesStorage: true, providesAnalytics: true })],
    }).compileComponents();

    fixture = TestBed.createComponent(AuthLandingPage);
    component = fixture.componentInstance;
    fixture.detectChanges();
  }));

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
