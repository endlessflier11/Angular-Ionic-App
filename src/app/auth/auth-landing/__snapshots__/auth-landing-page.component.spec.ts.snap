// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthLandingPage should match snapshot 1`] = `
<app-csaa-auth-landing
  __ngContext__={[Function LRootView]}
  navController={[Function NavController]}
  route={[Function Function]}
>
  <ion-content>
    <div
      class="wrapper"
    >
      <ion-row>
        <ion-col>
          <ion-grid>
            <ion-item-group>
              <ion-button
                expand="block"
              >
                Sign Up
              </ion-button>
            </ion-item-group>
            <ion-item-group>
              <ion-text>
                OR
              </ion-text>
            </ion-item-group>
            <ion-item-group>
              <ion-button
                color="primary"
                expand="block"
                fill="outline"
                routerlink="sso"
                tabindex="0"
              >
                Log In (SSO)
              </ion-button>
            </ion-item-group>
          </ion-grid>
        </ion-col>
      </ion-row>
      <ion-row>
        <ion-col
          class="ion-text-center"
        >
          <ion-button
            fill="none"
          >
            Back
          </ion-button>
        </ion-col>
      </ion-row>
    </div>
  </ion-content>
</app-csaa-auth-landing>
`;
